I do have issues, I have few instances of something like
```
FAILED: [code=1] src/doc/inventorymisc 
/home/portage/sci-mathematics/sagemath-doc-9999/temp/python3.13/bin/python3 /home/portage/sci-mathematics/sagemath-doc-9999/work/sagemath-doc-9999/src/build-docs.py --no-prune-empty-dirs --no-pdf-links reference/misc inventory -o src/doc --source /home/portage/sci-mathematics/sagemath-doc-9999/work/build_doc/src/doc
[misc     ] building [inventory]: targets for 100 source files that are out of date
[misc     ] updating environment: [new config] 100 added, 0 changed, 0 removed
[misc     ] /usr/lib/python3.13/site-packages/sphinx/events.py:394: DeprecationWarning:
[misc     ] Importing benchmark from here is deprecated; please use "from sage.misc.benchmark import benchmark" instead.
[misc     ] See https://github.com/sagemath/sage/issues/34259 for details.
[misc     ]   repr_args = repr(args)
[misc     ] WARNING: error while formatting arguments for sage.misc.c3_controlled.HierarchyElement.bases:  [autodoc]
[misc     ] WARNING: error while formatting arguments for sage.misc.c3_controlled.HierarchyElement.cls:  [autodoc]
[misc     ] WARNING: error while formatting arguments for sage.misc.c3_controlled.HierarchyElement.mro:  [autodoc]
[misc     ] WARNING: error while formatting arguments for sage.misc.c3_controlled.HierarchyElement.mro_controlled:  [autodoc]
[misc     ] WARNING: error while formatting arguments for sage.misc.c3_controlled.HierarchyElement.mro_standard:  [autodoc]
[misc     ] The inventory file is in src/doc/inventory/en/reference/misc.
Error building the documentation.
Traceback (most recent call last):
  File "/home/portage/sci-mathematics/sagemath-doc-9999/work/sagemath-doc-9999/src/build-docs.py", line 11, in <module>
    main()
    ~~~~^^
  File "/home/portage/sci-mathematics/sagemath-doc-9999/work/sagemath-doc-9999/src/sage_docbuild/__main__.py", line 548, in main
    build()
    ~~~~~^^
  File "/home/portage/sci-mathematics/sagemath-doc-9999/work/sagemath-doc-9999/src/sage_docbuild/builders.py", line 669, in _wrapper
    getattr(DocBuilder, build_type)(self, *args, **kwds)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/portage/sci-mathematics/sagemath-doc-9999/work/sagemath-doc-9999/src/sage_docbuild/builders.py", line 142, in f
    runsphinx()
    ~~~~~~~~~^^
  File "/home/portage/sci-mathematics/sagemath-doc-9999/work/sagemath-doc-9999/src/sage_docbuild/sphinxbuild.py", line 324, in runsphinx
    sys.stderr.raise_errors()
    ~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/home/portage/sci-mathematics/sagemath-doc-9999/work/sagemath-doc-9999/src/sage_docbuild/sphinxbuild.py", line 255, in raise_errors
    raise OSError(self._error)
OSError: WARNING: error while formatting arguments for sage.misc.c3_controlled.HierarchyElement.bases:  [autodoc]
```
Maybe I do something wrong or I am missing something?

--- a/src/doc/meson.build
+++ b/src/doc/meson.build
@@ -35,7 +35,7 @@ doc_bootstrap = custom_target(
   # doc_src is not really a dependency of the bootstrap, but we want to make sure
   # that all the source files are present before running the actual doc build
   # so let's collect all source-related targets here.
-  depends: doc_src
+  depends: doc_src,
 )
 
 references = run_command(

